modules/ROOT/pages/introduction/intro-config-use-studio.adoc

```diff
@@ -0,0 +1,160 @@
+= Use Studio to Configure a Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To use Anypoint Studio to configure a connector:
+
+. Supply your authentication credentials to Anypoint Platform.
+. Install the connector.
+. Configure an input source for the connector's flow.
+. Configure the connector.
+
+Follow these best practices when you configure an input source and connector in Studio:
+
+*  Configure an associated global element for the input source and connector.
+* Create a YAML file (`.yaml`) or properties file (`.properties`) to contain your properties, and then reference the file from the configuration.
+* Configure a reconnection strategy for the input source and target connector.
+
+For information about these best practices, see <<introduction/intro-connector-configuration-overview.adoc#best-practices,Best Practices for Configuring Connectors>>.
+
+== Supply Authentication Credentials to Anypoint Platform
+
+To access connectors on the Exchange portal, you must first configure Anypoint Platform credentials if these have not been configured:
+
+* If you use the US cloud, see xref:studio::set-credentials-in-studio-to.adoc[Configuring Anypoint Platform Credentials (Anypoint Studio)] for information.
+* If you use the EU cloud or Private Cloud Edition (PCE), see xref:studio::pce-configuration.adoc[Configure Private Cloud Edition in Studio] for information.
+
+[[install]]
+== Install the Connector
+
+Installing a connector adds it to a Studio project:
+
+. In Studio, create a Mule project.
+. In the Mule Palette view, click *(X) Search in Exchange*.
+. In *Add Dependencies to Project*, enter the connector name in the search field.
+. Click the connector name in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+[[configureInputSource]]
+== Configure an Input Source for the Connector
+
+An input source initiates a flow when a specified condition is met. Configure an input source for the connector's flow, such as an HTTP Listener operation (HTTP Listener), Scheduler endpoint (Scheduler), or connector-specific input source.
+
+=== Use HTTP Listener as an Input Source
```

Comment:
Why are we putting so many procedures in a concept topic? I thought this was an overview. These detailed procedures are already in the doc for the specific connectors.

Reply:
This topic is not the overview topic. It is called Use Studio to Configure a Connector.